src/gatling/simulations/uk/gov/hmcts/reform/exui/performance/scenarios/EXUIIACMC.scala

```diff
@@ -63,19 +108,14 @@ object EXUIIACMC {
     .group("XUI${service}_050_StartCreateCase1") {
       exec(http("XUI${service}_050_005_StartCreateCase1")
         .get("/data/internal/case-types/Asylum/event-triggers/startAppeal?ignore-warning=false")
-        .headers(IACHeader.headers_startcreatecase)
+        .headers(Headers.commonHeader)
+        .header("accept", "application/vnd.uk.gov.hmcts.ccd-data-store-api.ui-start-case-trigger.v2+json;charset=UTF-8")
         .check(status.is(200))
         .check(jsonPath("$.event_token").optional.saveAs("event_token")))
```

Comment:
I don't think this should be optional, as it's a necessity to proper function of later steps